cmd/stellar-rpc/internal/daemon/daemon.go

```diff
@@ -122,6 +122,16 @@ func (d *Daemon) Close() error {
 
 // newCaptiveCore creates a new captive core backend instance and returns it.
 func newCaptiveCore(cfg *config.Config, logger *supportlog.Entry) (*ledgerbackend.CaptiveStellarCore, error) {
+	var queryServerParams *ledgerbackend.HTTPQueryServerParams
+	if cfg.CaptiveCoreHTTPPort != 0 {
```

Comment:
wrong one, should be `CaptiveCoreHTTPQueryPort`

Reply:
I wish I had read this before debugging it :)